src/vr/clock.zig

```diff
@@ -423,8 +423,8 @@ fn formatDurationSigned(
 }
 
 const testing = std.testing;
-const OffsetType = @import("../test/time.zig").OffsetType;
```

Comment:
These clock changes are also an artifact from an earlier `beta2`. Would you be able to get them back to where they are now?

Reply:
Especially how we reworked the network options (see below), since we changed how the PacketSimulator handles path directions.